test/check-storage-reclaim

```diff
@@ -343,7 +343,6 @@ class TestStorageReclaimExistingSystemFedora(VirtInstallMachineCase):
         i.open()
         i.reach(i.steps.INSTALLATION_METHOD)
         s.set_scenario("use-free-space")
-        s.reclaim_set_checkbox(True)
```

Comment:
It's hard for me to review that -- How does that checkbox relate to increasing the disk image size?

Reply:
The checkbox is automatically set to checked and is disabled, is there is not enough space for the new installation on the disk. This payload image refresh, makes the new installation require a few more MB than the previous payload, making the free space on the image not enough for the new installation, therefore the checkbox is now disabled.